scripts/version-check.sh

```diff
@@ -26,6 +27,17 @@ else
     exit 1
 fi
 
+echo "▸ Validating ${CARTHAGE_JSON_SPECIFICATION}"
+
+cat $CARTHAGE_JSON_SPECIFICATION | jq -e --arg release_version "$release_version" 'has($release_version)'
```

Comment:
praise: 😍 so much good validation here!

Reply:
Thanks, these help move releases through the testing process quicker. The plan is to progressively script as many manual checks over time as feasible.